### ISSUE TITLE
type suspend with duration doesn't resume after passed the duration when working with subSteps

### ISSUE THREAD

**Issue by ShirokumaBearHouse** [kubevela/kubevela#4508]
**Describe the bug**
type suspend with duration doesn't resume after passed the duration when working with subSteps

**To Reproduce**
1. Having type suspend inside a subSteps with a duration

**Expected behavior**
To resume workflow after pass the duration periods

**KubeVela Version**
v1.4.3

**Example**
```
spec:
  components: []
  workflow:
    steps:
        - name: update
          subSteps:
            - name: wait-update
              properties:
                duration: 10s
              type: suspend
           type: step-group
status:
   status: workflowSuspending
   workflow:
    finished: false
    message: suspended
    mode: StepByStep
    startTime: "2022-07-29T11:42:58Z"
    steps:
    - firstExecuteTime: "2022-07-29T11:42:58Z"
      id: ql55pesi6q
      lastExecuteTime: "2022-07-29T11:42:58Z"
      name: update
      phase: running
      subSteps:
      - firstExecuteTime: "2022-07-29T11:42:58Z"
        id: 2adzqayi5d
        lastExecuteTime: "2022-07-29T11:42:58Z"
        name: wait-update
        phase: running
        type: suspend
      type: step-group
    suspend: true
    terminated: false   
```

**Comment by FogDong** [comment]
Thanks for the report, this is fixed in v1.5.0. Please try to upgrade your vela version and give it a try.

**Comment by wonderflow** [comment]
closing as it's fixed in v1.5.0 and this version is alread released. Feel free to reopen it if there's still any issues.
